### PR TITLE
Edited log message to only show failed deletes

### PR DESCRIPTION
**Pull request by tlangs**
### Description
Made the log message for a failed move only show the files that haven't been deleted.
----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: Ensure that you have added or updated tests
- [ ] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

File: clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutor.scala [hunk 1]
```diff
@@ -177,10 +177,11 @@ class MoveExecutor[TI <: TransferIndex](moveCommand: MoveCommand[TI])
           .sequence(googleDeletes)
           .recover {
             case ex =>
+              val notDeleted = oldPaths.par.filter(ioUtil.googleObjectExists).seq
```
Comment (danxmoran) [inline clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutor.scala]: We shouldn't need to hit the network again, we can already get the info of what failed by being smarter about how we collect all the delete `Future`s.

Comment (tlangs) [reply clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutor.scala]: I'll give that a go. Probably have to transform to a sequence of future futures.

**Review by danxmoran — APPROVED** [review]

File: clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutor.scala [hunk 2]
```diff
@@ -175,16 +183,19 @@ class MoveExecutor[TI <: TransferIndex](moveCommand: MoveCommand[TI])
...
+              val notDeleted = moves.filter(_.isLeft)
               throw new RuntimeException(
                 s"""The old files associated with $prettyKey were not able to be deleted.
-                   |Please manually delete the old files at:
-                   |${oldPaths.mkString(",\n")}
-                   |If this can't be done, please contact the Green Team at ${ClioClientConfig.greenTeamEmail}""".stripMargin,
-                ex
+                 |Please manually delete the old files at:
+                 |${notDeleted.mkString(",\n")}
+                 |If this can't be done, please contact the Green Team at ${ClioClientConfig.greenTeamEmail}""".stripMargin
```
Comment (tbl3rd) [inline clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutor.scala]: If you cannot delete the files, ...
OR
Please tell the Green Team at ... if you cannot delete the files.

**Review by tbl3rd — APPROVED** [review]
Maybe a more concrete error message.

**Pull request merged**